astoria/common/manager_requests.py

```diff
@@ -0,0 +1,34 @@
+"""Schema definitions for manager requests."""
+from uuid import UUID, uuid4
+
+from pydantic import BaseModel, Field
+
+
+class ManagerRequest(BaseModel):
+    """Schema definition for a Manager Request."""
+
+    uuid: UUID = Field(default_factory=uuid4)
+    sender_name: str  # client name of component that sent the request
+
+
+class RequestResponse(BaseModel):
+    """
+    Schema definition for the response to a Manager Request.
+
+    Do not extend or subclass. The response should always be of identical format.
```

Comment:
the `typing.final` decorator looks useful here (it's 3.8+ only, but it looks like it's in typing-extensions)

Reply:
3600173